DEPENDENCIES.rst

```diff
@@ -55,8 +57,8 @@ Package                Minimum Version
 ====================   ==================
 doxygen (for docs)     1.7.6.1
 tcmalloc (for speed)   any
-Cython                  >=0.25 and  <0.27
-Python (dev version)   2.7 or 3.3+
+Cython                 0.29+
```

Comment:
I thought we were updated to Cython 3 - (@bennibbelink) but maybe I missed that we are still compatible with cython 0.29?

Reply:
We are still compatible with cython 0.29.  Installing `cython3` via `apt` installs version 0.29 in both Ubuntu 20.04 and 22.04